samples/bluetooth/central_hr/src/main.c

```diff
@@ -155,11 +276,11 @@ static bool eir_found(struct bt_data *data, void *user_data)
 				printk("Stop LE scan failed (err %d)\n", err);
 				continue;
 			}
-
```

Comment:
Don't remove this line. In general always an empty line after `}` unless the next line is also `}` (just to keep the style for all Bluetooth code consistent). I think you had some places in this PR that should be fixed.